<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LyricWidget</class>
 <widget class="QWidget" name="LyricWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="SingleLyricFrame" name="frameLyric">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameControl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>24</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnPrevios">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>上一首</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res.qrc">
                 <normaloff>:/icons/images/previous_lyric.png</normaloff>:/icons/images/previous_lyric.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPlay">
               <property name="minimumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>播放</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res.qrc">
                 <normaloff>:/icons/images/play_lyric.png</normaloff>:/icons/images/play_lyric.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNext">
               <property name="minimumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>下一首</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../res.qrc">
                 <normaloff>:/icons/images/next_lyric.png.png</normaloff>:/icons/images/next_lyric.png.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnClose">
             <property name="minimumSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>关闭</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/icons/images/close_lyric.png</normaloff>:/icons/images/close_lyric.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>14</width>
               <height>14</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="SingleLyricLabel" name="labelLyric">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>24</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleLyricLabel</class>
   <extends>QLabel</extends>
   <header>singlelyriclabel.h</header>
  </customwidget>
  <customwidget>
   <class>SingleLyricFrame</class>
   <extends>QFrame</extends>
   <header location="global">singlelyricframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
